esp/logging/logginglib/LogFailSafe.cpp

```diff
@@ -89,6 +92,7 @@ void CLogFailSafe::readCfg(IPropertyTree* cfg)
         m_logsdir.set(logsDir);
     else
         m_logsdir.set(DefaultFailSafeLogsDir);
+    decoupledLogging = cfg->getPropBool("DecoupledLogging", false);
```

Comment:
I'm surprised to see the string literal for the property name, given the use of named values elsewhere.